jxl/src/render/stages/chroma_upsample.rs

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+use crate::render::{RenderPipelineInOutStage, RenderPipelineStage};
+
+pub struct NearestNeighbourUpsample {
```

Comment:
This doesn't *quite* fit here anymore if we call it chroma_upsample...

Reply:
ok, let's move nearest neighbor to just upsample.rs? or should we make a file nearest_neighbor.rs?